src/components/Settings/Form.js

```diff
@@ -41,11 +41,7 @@ const SettingsForm = () => {
   };
 
   return (
-    <Form
-      formMethods={formMethods}
-      className={styles.settingsForm}
-      onSubmit={onSubmit}
-    >
+    <FormBody formMethods={formMethods} onSubmit={onSubmit}>
```

Comment:
FormBody is not completely accurate right? considering it's actually the form itself. maybe we can pick a better name for this?

Reply:
Form and FormSettings are taken. FormContainer :P

Reply:
what if you just import { Input, Select } from the form? you are not using the `Form` anymore here

Reply:
to do that we should export `Input` and `Select` from `Form`, which are currently static properties. @mollerjorge what do you think about exporting them?

Reply:
What about reusing the form and instead of creating a new FormBody like we did here we just add an specific className to the Form component instance like this:

```
<Form className={someClassNameSpecificToSettingsToOverrideFormStyles} >
   <Form.Input/>
</Form>
```

Refer to this sandbox on how we can do it https://codesandbox.io/s/emotion-styled-ggn6x?file=/src/index.js:386-398